File: medplum-java/medplum-model/src/main/java/com/fhir/ValueSet_Include.java
package com.fhir;

import com.StagedBuilderStyle;
import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
import java.lang.String;
import java.util.List;
import java.util.Optional;
import org.immutables.value.Value;

@Value.Immutable
@StagedBuilderStyle
@JsonDeserialize(
    as = ImmutableValueSet_Include.class
)
@JsonIgnoreProperties(
    ignoreUnknown = true
)
public interface ValueSet_Include {
  Optional<String> id();

  Optional<uri> system();

  Optional<String> version();

  Optional<List<ValueSet_Concept>> concept();

  Optional<List<Extension>> extension();

  Optional<List<ValueSet_Filter>> filter();

  Optional<List<canonical>> valueSet();

  Optional<List<Extension>> modifierExtension();

  static ImmutableValueSet_Include.Builder builder() {
    return ImmutableValueSet_Include.builder();
  }
}

File: medplum-typescript/src/model/fhir/ValueSet_Include.ts
import { canonical } from "./canonical";
import { Extension } from "./Extension";
import { uri } from "./uri";
import { ValueSet_Concept } from "./ValueSet_Concept";
import { ValueSet_Filter } from "./ValueSet_Filter";

export interface ValueSet_Include {
    id: string | null | undefined;
    extension: Extension[] | null | undefined;
    modifierExtension: Extension[] | null | undefined;
    system: uri | null | undefined;
    version: string | null | undefined;
    concept: ValueSet_Concept[] | null | undefined;
    filter: ValueSet_Filter[] | null | undefined;
    valueSet: canonical[] | null | undefined;
}
